requirements-base.txt

```diff
@@ -105,5 +84,4 @@ setproctitle>=1.1.7,<1.2.0
 phabricator>=0.6.0,<0.8
 
 # test dependencies, but unable to move to requirements-test until sentry.utils.pytest and similar are moved to tests/
-mock==3.0.5 ; python_version < "3.3"
```

Comment:
@BYK You missed `src/sentry/utils/compat/mock.py`, but I also see a few direct `import mock`s.... how are those not failing? I can fail it locally:

```
tests/sentry/mail/test_adapter.py:7: in <module>
    import mock
E   ModuleNotFoundError: No module named 'mock'
```

Reply:
Huh, `mock`'s still being installed in CI. But not the others. It must be a transient dependency, I just directly pip uninstalled mock from my dev env. Lemme try the whole thing.